tests/App/cache/prod/AppProdDebugProjectContainer.php

```diff
@@ -0,0 +1,17 @@
+<?php
```

Comment:
@Jean85 This file from cache and all the other cache files should have not been commited.

Reply:
are we missing a .gitignore here?

Reply:
Yes, the `.gitignore `file was updated after the commit with all the cache files: https://github.com/liip/LiipFunctionalTestBundle/pull/359/commits/ec55e5c4ecce7320e1e02aef216ae5e432654d53